dependencies/src/test/java/com/google/cloud/tools/opensource/dependencies/RepositoryUtilityTest.java

```diff
@@ -63,4 +64,23 @@ public void testReadBom_path() throws MavenRepositoryException {
     Assert.assertTrue(coordinates.startsWith("com.google.cloud:libraries-bom:"));
     Assert.assertTrue(coordinates.endsWith("-SNAPSHOT"));
   }
+
+  @Test
+  public void testFindVersions() throws MavenRepositoryException {
+    RepositorySystem system = RepositoryUtility.newRepositorySystem();
+    ImmutableList<String> versions =
+        RepositoryUtility.findVersions(system, "com.google.cloud", "libraries-bom");
+    Truth.assertThat(versions).containsAtLeast("1.1.0", "1.1.1", "1.2.0", "2.0.0").inOrder();
+  }
+
+  @Test
+  public void testFindHighestVersions() throws MavenRepositoryException {
+    RepositorySystem system = RepositoryUtility.newRepositorySystem();
+    String guavaHighestVersion =
+        RepositoryUtility.findHighestVersion(
+            system, RepositoryUtility.newSession(system), "com.google.guava", "guava");
+    Assert.assertNotNull(guavaHighestVersion);
+    // Latest guava release is greater than or equal to 28.0.
+    Truth.assertThat(guavaHighestVersion).isAtLeast("28.0");
```

Comment:
This is doing case sensitive string comparisons. I'm not sure that's what you want.

Reply:
Good catch; it will fail when guava's "100.0-jre" release. Updated.